### PR TITLE
fix(unlock-app): showing time in the event's tz

### PR DESCRIPTION
**Pull request by julien51**
# Description

As requested by @ccarfi !

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/components/content/event/EventDetails.tsx [hunk 1]
```diff
@@ -139,7 +139,9 @@ export const EventDetails = ({ lockAddress, network }: EventDetailsProps) => {
           {eventDate && eventData.ticket?.event_start_time && (
             <li className="mb-2">
               <FaClock className="inline mr-2" />
-              {eventDate.toLocaleTimeString()}
+              {eventDate.toLocaleTimeString(navigator.language || 'en-US', {
+                timeZone: eventData.ticket.event_timezone,
+              })}
```
Comment (searchableguy) [inline unlock-app/src/components/content/event/EventDetails.tsx:144]: Do we hint that we are using event location timezone than user timezone? If not, we should.

Comment (julien51) [reply unlock-app/src/components/content/event/EventDetails.tsx:144]: Good point. Let me add a hint!

**Comment by julien51** [comment]
Screenshot 2023-02-10 at 2 01 43 PM

**Comment by julien51** [comment]
PTAL @searchableguy

**Comment by julien51** [comment]
Added a tooltip which shows on mouse over :)

**Review by searchableguy — APPROVED** [review]
Approving otherwise.

**Pull request merged**